README_ENGLISH.md

```diff
@@ -0,0 +1,58 @@
+# Project Buy It
+[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=salomaoluiz_BuyIt&metric=alert_status)](https://sonarcloud.io/dashboard?id=salomaoluiz_BuyIt)
+
+This project has be created as a study of case several at development points. If you want use this project to study or commercially, feel free, as long as you follow the [LICENSE](https://github.com/salomaoluiz/BuyIt/blob/main/LICENSE) rules. Most of this project has the descriptions in portuguese, because this way is more easily to a brazilian starter developer understant without language trouble, just the code is fully in english.
+
+## Firebase
+For this project is used the Firebase as a BAS (Backend as a Service), soo you need realize some settings in your project before start.
+
+First, you need create a project in Firebase (Two projects if you wanna separe the development environment of production environment), configure the project to Android and add the **google-services.json** in folders *./android/app/src/development* and *./android/app/src/production*.
+
+Realized this initial settings, you need adjust the ADMOB for the ads, to configure this is required create 3 files, with each one theys with the following informations:
+
+- **firebase.json**
+```
+{
+  "react-native": {
+    "admob_android_app_id": "APP_ID_DO_ADMOB_ANDROID",
+    "admob_ios_app_id": "APP_ID_DO_ADMOB_IOS"
+  }
+}
+```
+- **.env.dev**
+```
+DEFAULT_ENVIRONMENT=development
+ADMOB_APP_ID=ADMOB_APP_ID
+ADMOB_BANNER_ID_DEFAULT=ADMOB_BANNER_ID_DEFAULT
+ADMOB_BANNER_ID_CAMPAING=
+ADMOB_INTERSTITIAL_ID_DEFAULT=ADMOB_INTERSTITIAL_ID_DEFAULT
+ADMOB_INTERSTITIAL_ID_CAMPAING=
+```
+
+- **.env.prod**
+```
+DEFAULT_ENVIRONMENT=production
+ADMOB_APP_ID=ADMOB_APP_ID
+ADMOB_BANNER_ID_DEFAULT=ADMOB_BANNER_ID_DEFAULT
+ADMOB_BANNER_ID_CAMPAING=
+ADMOB_INTERSTITIAL_ID_DEFAULT=ADMOB_INTERSTITIAL_ID_DEFAULT
+ADMOB_INTERSTITIAL_ID_CAMPAING=
+```
+
+Look, in spite of has a `ADMOB BANNER _ID_*` in the development environment, while you are using the project without release mode, it will show only the test ads.
```

Comment:
```suggestion
> **Note:**, In spite having `ADMOB BANNER _ID_*` in the development environment, while you are using the project in development mode, it will only show test ads.
```